pkg/errors/errors.go

```diff
@@ -0,0 +1,14 @@
+package errors
+
+import (
+	"fmt"
+	"strings"
+)
+
+func newInvalidCRD(apiGroup string) *KNError {
+	parts := strings.Split(apiGroup, ".")
```

Comment:
I would like some feedback on this approach, I am using the first item in the resource group as the "api name"